tests/e2e/specs/upe-split/shopper/shopper-upe-enabled-all-flows.spec.js

```diff
@@ -48,73 +36,6 @@ describe.skip( 'Enabled Split UPE', () => {
 		await merchant.logout();
 	} );
 
-	describe( 'Blocks checkout', () => {
```

Comment:
What was the reason for removing the Blocks Checkout tests from the Split UPE? Are these tests covered somewhere else?

Reply:
All Blocks Checkout tests are broken/failing on `develop` at the moment, so it seems that tests involving Blocks are removed or otherwise suppressed. This is mentioned in [this thread](p1676324326527639/1676321242.264789-slack-C048UEB37MZ). I think this is the reason for the `SKIP_WC_BLOCKS_TESTS` flag as well.

Reply:
Thanks for clarifying! Could you please log an issue for [the post-rollout epic](https://github.com/Automattic/woopay/issues/1631) to ensure we come back to this and fix the tests on the blocks plugin?